k8s/graphscope.Dockerfile

```diff
@@ -97,6 +101,11 @@ RUN source ~/.bashrc \
            echo "debug mode" && ./exec.sh cargo build --all; \
        fi
 
+# gie frontend / coordinator
+RUN cd ${HOME}/gs/interactive_engine \
+    && mvn clean package -DskipTests -Pjava-release --quiet
+
+
 # # # # # # # # # # # # # # # # # # # # # #
 # generate final runtime image
 FROM registry.cn-hongkong.aliyuncs.com/graphscope/graphscope-runtime:latest
```

Comment:
Add `SHELL ["/usr/bin/scl", "enable", "devtoolset-7"]` here as well.

I think the final image also needs that.